gamemode/modules/rafts/sv_commands.lua

```diff
@@ -0,0 +1,97 @@
+local function summonCommandCallback( ply )
+    local trace = util.TraceLine{
+        start  = ply:EyePos(),
+        endpos = ply:EyePos() + ply:EyeAngles():Forward() * 1000,
+        mask   = MASK_ALL,
+        filter = ply,
+    }
+
+    local pos = trace.HitPos
+    local raft = RVR.createRaft( pos )
+    raft:AddOwnerID( ply:SteamID() )
+end
+
+local function placeWallCallback( ply, piece, class, yaw )
+    local raft = piece:GetRaft()
+    local clsTable = baseclass.Get(class)
+
+    if not raft then return end
+
+    if not raft:CanBuild( ply ) then
+        ply:PrintMessage( HUD_PRINTCONSOLE, "you do not hav epermission to build on this raft" )
+        return
+    end
+    
+    local required = clsTable:GetRequiredItems()
+
+    local success, itemsMissing = RVR.Inventory.checkItems( ply.RVR_Inventory, required )
+    if not success then
+        -- TODO print itemsMissing in a human readable format
+        ply:PrintMessage( HUD_PRINTCONSOLE, "missing required items" )
+        return
+    end
+
+    local _,  err = RVR.placeWall( piece, class, yaw )
+    if err ~= nil then
+        return "Couldn't place  wall: " .. err
+    end
+
+    local success, itemsMissing = RVR.Inventory.tryTakeItems( ply, required )
+end
+
+local function expandCallback( ply, piece, class, x, y ,z, yaw )
+    local raft = piece:GetRaft()
+    if not raft then return end
+
+    local clsTable = baseclass.Get(class)
+
+    if not raft:CanBuild( ply ) then
+        ply:PrintMessage( HUD_PRINTCONSOLE, "you do not have permissions to build on this raft" )
```

Comment:
Same as above, though this should probably be pull out into localization *but that ain't merged yet* 🙃 
```suggestion
        ply:PrintMessage( HUD_PRINTCONSOLE, "You do not have permissions to build on this raft" )
```